### PR TITLE
zile 2.6.0.90

### PR DESCRIPTION
**Pull request by carlocab**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]

**Comment by carlocab** [comment]
@chenrui333 this didn't get merged for some reason...
```
❯ brew livecheck zile
zile : 2.4.15 ==> 2.6.0.90
```

**Comment by carlocab** [comment]
Oh, I see. I closed the pull request by mistake.

**Comment by chenrui333** [comment]
yeah, looks like it 🤦

**Comment by carlocab** [comment]
I don't know how I did that though. 😅 I think it happened automatically when I deleted the branch on my fork after the PR was approved.

**Comment by chenrui333** [comment]
yeah, if you deleted the branch, then PR for sure would be auto-closed.

**Comment by carlocab** [comment]
That seems silly, and GitHub should at least warn me that that would happen when I did it.

**Comment by carlocab** [comment]
Ok; it's safe to delete my branch now, right?? 😆

File: Formula/zile.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 class Zile < Formula
   desc "Text editor development kit"
   homepage "https://www.gnu.org/software/zile/"
-  url "https://ftp.gnu.org/gnu/zile/zile-2.4.15.tar.gz"
-  mirror "https://ftpmirror.gnu.org/zile/zile-2.4.15.tar.gz"
-  sha256 "39c300a34f78c37ba67793cf74685935a15568e14237a3a66fda8fcf40e3035e"
+  url "https://ftp.gnu.org/gnu/zile/zile-2.6.0.90.tar.gz"
+  mirror "https://ftpmirror.gnu.org/zile/zile-2.6.0.90.tar.gz"
```
Comment (chenrui333) [inline Formula/zile.rb:5]: looks like i missed this earlier. 

For gnu stuff, the `9x` releases stand for beta release, and sometimes, odd number release stands for beta releases as well.

Comment (carlocab) [reply Formula/zile.rb:5]: Should be fine now. The author says they'll no longer publish betas to https://ftp.gnu.org/gnu/zile, so I don't think we need to worry about this anymore. That said, I still left the comment to check that the tarball is not a pre-release.

**Pull request closed without merging**